### PR TITLE
Add AngelDAO theme

### PR DESCRIPTION
**Pull request by meirbank**
**If you want to create a space follow this guide**

https://docs.snapshot.page/guides/create-a-space

### CODE REVIEW

File: skins/angeldao.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.benchmark {
```
Comment (gokaiorg) [inline skins/angeldao.scss:1]: The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.angeldao {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your https://snapshot.page/#/angeldao.eth space, did you create it already?

**Comment by meirbank** [comment]
> I can't find your https://snapshot.page/#/angeldao.eth space, did you create it already?

No we will create it soon, need to get our multi sig wallet ready. Thanks for the fix.

**Comment by gokaiorg** [comment]
@meirbank Please let me know when your new space is live then i will merge this PR. Thank you.

**Comment by meirbank** [comment]
Created, https://snapshot.page/#/AngelDAO.eth

Thanks

**Comment by gokaiorg** [comment]
@bonustrack Does an ENS with uppercase letters need a folder name for the space with uppercase letters too? Thank you.

**Comment by bonustrack** [comment]
@meirbank @gokaiorg Hi sorry but both the folder and the ENS domain should be lowercased.

**Comment by meirbank** [comment]
Ok we will have to make a new snapshot page soon then I guess

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**